### DISCUSSION TITLE
any method to rename code-server

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5635] [General] [answered]
sometimes, I want to customize the code-server, for example, I hope to customize the web title, it's not code-server, just as x-ide and so on.

I want to know any method to do it.

<img width="645" alt="image" src="https://user-images.githubusercontent.com/9245110/194797545-85451e23-e702-4717-85aa-3a2d64bccf1e.png">

If there is no direct method, I think I can only modify the source code to package it myself.

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
I think you have to modify the source code. The DevTools settings are temporary; editing the source code and repackaging makes the change permanent.

**Reply by alanhe421** [reply]
maybe coder-server support app-name config.

**Reply by lg-nineoneninethreenine** [reply]
@jsjoeio does it?

**Reply by jsjoeio** [reply]
I don't think so. I think @im-coder-lg is right in that you'd have to modify the source code. I believe this comes from the `product.json` which we modify as part of the build here: https://github.com/coder/code-server/blob/main/ci/build/build-vscode.sh#L51-L91
